hydroshare/urls.py

```diff
@@ -43,7 +45,8 @@
     url(r'^django_irods/', include('django_irods.urls')),
     url(r'^django_docker_processes/', include('django_docker_processes.urls')),
     url(r'^autocomplete/', include('autocomplete_light.urls')),
-    url(r'^search/$', FacetedSearchView(form_class=MyForm), name='haystack_search'),
+    url(r'^search/$', DiscoveryView.as_view(), name='haystack_search'),
+    url(r'^searchjson/$', DiscoveryJsonView.as_view(), name='haystack_json_search'),
```

Comment:
recommend putting the search api view under `search/` such as `search/api/` or `search/json`